api/paidAction/index.js

```diff
@@ -244,28 +244,20 @@ export async function retryPaidAction (actionType, args, incomingContext) {
     throw new Error(`retryPaidAction - must be logged in ${actionType}`)
   }
 
-  if (!action.paymentMethods.includes(PAID_ACTION_PAYMENT_METHODS.OPTIMISTIC)) {
-    throw new Error(`retryPaidAction - action does not support optimism ${actionType}`)
-  }
-
-  if (!action.retry) {
-    throw new Error(`retryPaidAction - action does not support retrying ${actionType}`)
-  }
-
```

Comment:
So far, I think it's okay to remove these checks to allow retries for pessimistic actions but I will make more sure

Reply:
I see no problem with removing these. Retries for optimistic actions work like before and pessimistic actions support retries as expected now.